listenbrainz/db/user.py

```diff
@@ -164,3 +165,25 @@ def get_or_create(musicbrainz_id):
         create(musicbrainz_id)
         user = get_by_mb_id(musicbrainz_id)
     return user
+
+def update_last_login(musicbrainz_id, ts=int(time.time())):
```

Comment:
Is there ever a case where we want to set the last login time to a time that isn't now? If not, I would consider removing the argument and making this always set the last_login to the current time. As @mayhem suggested, it would be better to use the database to set this (`SET last_login = now()`)